recipes/newrelic/infrastructure/ohi/mongodb/windows.yml

```diff
@@ -0,0 +1,74 @@
+name: mongodb-open-source-integration
+displayName: MongoDB Open Source Integration
+description: New Relic install recipe for default MongoDB Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-mongodb
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: windows
+    kernelArch: x86_6
```

Comment:
@NRhzhao I've tested the recipe, and I'm not seeing the DETECTED event for MongoDB on Windows.
I think this kernelArch version is incorrect.